app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity/activity_case_cancellation.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.request.AuthenticatedRequest
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{CancelReason, CancellationCaseStatusChange, Event}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.progressive_disclosure
+@(index: Int, event: Event)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+
+@content(body: CancellationCaseStatusChange) = {
+ <p>
+  <span class="bold">Reason:</span>
```

Comment:
Is the content right here? Can't see anywhere on the prototypes/designs where we should show **Reason**: - it seems to be like that for note, but the other stuff seems to be styled just the reason itself in bold, e.g **Invalidated due to EU measure (62)**

Reply:
I couldn't find any screenshots for it, let's see if they say anything at post amigos